### DISCUSSION TITLE
Error _cyrfc DLL when try import pyrfc

### DISCUSSION THREAD

**Discussion by eumateuscandido** [SAP/PyRFC#317] [Q&A]
When a type the command on Jupyter Notebook, I facing the error:

ImportError: DLL load failed while importing _cyrfc: The specified module could not be found.

Python Version: 3.11
SAP SDK Folders: C:\Users\pc\Documents\Tools\nwrfcsdk
SAPNWRFC_HOME set variable

![image](https://user-images.githubusercontent.com/80724923/235231231-4ad33639-2180-4610-bc8a-535f0ab52110.png)

LIB setting in variable path
![image](https://user-images.githubusercontent.com/80724923/235231727-9c2b567b-1ab5-4d35-a98f-a1b5cca83a61.png)

I already try update the python, set the variable path from other folders but just dosen't work,

**Comment by bsrdjan** [comment]
Is it still not working?

**Comment by Emmanuehl** [comment]
I'm having the same problem, even after making sure I didn't miss a step or something. 
![image](https://github.com/SAP/PyRFC/assets/69962596/7886d59c-54a6-4cb1-ba13-11ee0006ca18)
I have the latest Python and the latest pyrfc
![image](https://github.com/SAP/PyRFC/assets/69962596/9914a749-97f3-4b7f-b600-8e8823ab9f7b)

**Reply by leandro2211** [reply]
I'm having exactly the same issue.

**Comment by bsrdjan** [comment]
Which SAP NW RFC SDK version is installed? To verify run `findstr Patch sapnwrfc.dll` 

Are Windows pre-requisites installed: https://github.com/SAP/PyRFC#windows ?
